tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -516,16 +516,6 @@ ifneq ($(OPTIMIZED_KERNEL_DIR),)
   MICROLITE_CC_KERNEL_SRCS := $(shell python3 $(MAKEFILE_DIR)/specialize_files.py \
 		--base_files "$(MICROLITE_CC_KERNEL_SRCS)" \
 		--specialize_directory $(PATH_TO_OPTIMIZED_KERNELS))
-
-  # The first ifneq is needed to be compatible with make versions prior to 4.2
-  # which do not support .SHELLSTATUS. While make 4.2 was released in 2016,
-  # Ubuntu 18.04 only has version 4.1
-  ifneq ($(.SHELLSTATUS),)
-    ifneq ($(.SHELLSTATUS),0)
-      $(error Error with specialize_files.py $(MICROLITE_CC_KERNEL_SRCS))
-    endif
-  endif
-
```

Comment:
is this intentional?

Reply:
oh. No.  Thanks for catching this!  This is a mistake by copy.

Fixed now.